frontend/src/components/terms/terms.js

```diff
@@ -13,38 +13,36 @@ const Terms = () => {
         </div>
       </div>
       <div className="agreement">
-          <div class="row">
```

Comment:
Only class keyword changed required to className. No need for removing the classes.